scripts/Projects.psm1

```diff
@@ -10,7 +10,7 @@
 $Frameworks = @{}
 
 # .NET Framework versions for which we build.
-$Frameworks.NetFx = @("net452", "net461", "net472")
+$Frameworks.NetFx = @("net48")
```

Comment:
>net48 [](http://example.com/codeflow?start=23&length=5)

I need to explore this script further to make sure it's correct. in-progress. TBD

Reply:
I see this is closed, did you have any notes besides the inline comments on your exploration?

Reply:
set as "Active" for marmegh's comment for better visibility.

Reply:
Nope, the comments stand. Let me know if they're insufficiently clear.

Also, I should stop 'closing' items and 'resolve' them so others can close. Sorry for this lapse in process! I just did it to one of your notes as well, Shaopeng. :)